src/Adyen/Util/OpenInvoice.php

```diff
@@ -31,6 +31,13 @@ class OpenInvoice
     const AFTERPAY_DIRECTDEBIT_PAYMENT_METHOD = 'afterpay_directdebit';
     const AFTERPAYTOUCH_PAYMENT_METHOD = 'afterpaytouch';
 
+    const AFTERPAYTOUCH = "afterpaytouch";
+    const AFTERPAYTOUCH_AU = "afterpaytouch_AU";
+    const AFTERPAYTOUCH_CA = "afterpaytouch_CA";
+    const AFTERPAYTOUCH_NZ = "afterpaytouch_NZ";
+    const AFTERPAYTOUCH_US = "afterpaytouch_US";
+    const AFTERPAYTOUCH_POS = "afterpaytouch_pos";
+
     const RATEPAY_PAYMENT_METHOD = 'ratepay';
```

Comment:
does it maybe make sense if we deprecate these consts and rename them to just `RATEPAY` instead of `RATEPAY_PAYMENT_METHOD`?